docs/appendix/index.rst

```diff
@@ -0,0 +1,7 @@
+.. _appendix-index:
+
+Appendix
+========
+
+This appendix shows some useful resources for developers and curious people, even when the contents
+are not essential for using DiddiParser 2, that's why they are in the appendix.
```

Comment:
Thinking about this, we can move what's inside the wiki to the appendix. But that would be in a future PR.

Reply:
See #91.